calyx-py/calyx/py_ast.py

```diff
@@ -286,14 +397,26 @@ class Cell(Structure):
     comp: CompInst
     is_external: bool = False
     is_ref: bool = False
+    attributes: list[CellAttribute] = field(default_factory=list)
+    loc: Optional[int] = field(default_factory=PosTable.determine_source_loc)
 
     def doc(self) -> str:
+        # NOTE: adding external on the fly (instead of having the user add it to attributes)
+        # so that we can easily do this check
```

Comment:
I feel like this comment is unnecessary since it is mostly justifying the change you made and would not make much sense to someone who wasn't familiar with the old version of the code